Article/Article.js

```diff
@@ -3,24 +3,34 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.expandButton') ;
+    //close button  
+    this.closeButton = this.domElement.querySelector('.close');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.innerText = 'expand';
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', () => this.expandArticle());
+    //button listener
+    this.closeButton.addEventListener('click', () => this.exitArticle());
+  }
+  exitArticle(){
+    this.domElement.classList.toggle('article-exit');
+    console.log(this.domElement.classList);
   }
-
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
-
+    this.domElement.classList.toggle('article-open');
+    
   }
 }
 
 // START HERE: Select all classes named ".article" and assign that value to the articles variable
-let articles;
-
+let articles = document.querySelectorAll('.article');
+console.log(document.querySelectorAll('.article'));
```

Comment:
Remove console logs once you're done debugging (unless they have a good reason to be in production)! (Otherwise it can end up making a real mess in your code and in your console.)